app/models/embedded_ansible_worker/runner.rb

```diff
@@ -36,6 +38,17 @@ def setup_ansible
     _log.info("calling EmbeddedAnsible.start")
     EmbeddedAnsible.start
     _log.info("calling EmbeddedAnsible.start finished")
+
+    5.times do
```

Comment:
Does it make sense to have `start` do the `sleep until alive?` so `start` doesn't return when it's not yet alive?